internal/cli/uninstall.go

```diff
@@ -46,8 +46,21 @@ func (c *UninstallCommand) Run(args []string) int {
 	}
 
 	if !c.autoApprove {
-		c.ui.Output(strings.TrimSpace(autoApproveMsg), terminal.WithErrorStyle())
-		return 1
+		proceed, err := c.ui.Input(&terminal.Input{
+			Prompt: "Do you really want to uninstall the Waypoint server? Only 'yes' will be accepted to approve. ",
```

Comment:
Let's use a colon instead of a period to end this? like "Only 'yes' will be accepted to approve: "